calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx

```diff
@@ -0,0 +1,22 @@
+xx May 2023
+
+### What's new {#whats-new-3.17.0}
+
+#### New policy recommendations engine for namespace isolation
+{{prodname}} has added a new policy recommendations engine that automatically generates policies for namespace isolation within your cluster. [Policy recommendations](network-policy/recommendations/policy-recommendations.mdx).
```

Comment:
> automatically generates policies

If we were to say:

> automatically generates staged policies

Would that perhaps take concerns away that they take effect automatically?
cc: @tigeraphil